### ISSUE TITLE
Agents review feedbacks

### ISSUE THREAD

**Issue by YanetABUC** [reconness/reconness-frontend#81]
- [ ] In the delete confirmation popup, the text will be "Please, confirm the name of the Agent **<agent_name>** before delete it", where <agent_name> is the name of the agent to be deleted.
- [ ] Display a success or error notification after editing an agent, 
- [ ] While trying to delete multiple agents, the link "Delete Agents" will be in red color, so the user can notice it
- [x] When the list is in multiple delete mode, add a Cancel link near Delete Agents link, to restore the list to its original status and clear the delete selection
![image](https://user-images.githubusercontent.com/4720000/104818416-1d937200-57ed-11eb-8309-ff141810c4b3.png)

**Comment by YanetABUC** [comment]
![image](https://user-images.githubusercontent.com/4720000/105576997-60e56780-5d3c-11eb-967f-09bf17bc0030.png)

**Comment by gorums** [comment]
@YanetABUC **Delete Agents** link needs to be disable if not Agent is selected to be delete, only after we select one or more Agents to be delete then the link  **Delete Agents** is enabled

**Comment by YanetABUC** [comment]
![image](https://user-images.githubusercontent.com/4720000/106009335-d5424280-607d-11eb-93b3-157d30c57784.png)

![image](https://user-images.githubusercontent.com/4720000/106018754-800b2e80-6087-11eb-83fd-4af1c80df20c.png)

![image](https://user-images.githubusercontent.com/4720000/106018821-91543b00-6087-11eb-975c-299fc1cc41af.png)

![image](https://user-images.githubusercontent.com/4720000/106018861-9ca76680-6087-11eb-8431-a3cf9d81c214.png)

**Comment by YanetABUC** [comment]
> @YanetABUC **Delete Agents** link needs to be disable if not Agent is selected to be delete, only after we select one or more Agents to be delete then the link **Delete Agents** is enabled

@gorums I'll create a new ticket with this request
